app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java

```diff
@@ -48,14 +48,14 @@ public Troubleshooter(EventBus eventBus) {
     }
 
     /**
-     * Returns true iff the given issue is current active.
+     * Returns true if the given issue is current active.
```

Comment:
FYI I meant this as ["if and only if"](https://en.wikipedia.org/wiki/If_and_only_if), in case you run across it again. But it's also probably unnecessary for a boolean. :)